middleware/sendJSON.js

```diff
@@ -42,7 +43,10 @@ function sendJSONResponse(req, res, next) {
         if( err instanceof es.errors.RequestTimeout ){ statusCode = Math.max( statusCode, 408 ); }
         else if( err instanceof es.errors.NoConnections ){ statusCode = Math.max( statusCode, 502 ); }
         else if( err instanceof es.errors.ConnectionFault ){ statusCode = Math.max( statusCode, 502 ); }
-        else { statusCode = Math.max( statusCode, 500 ); }
+        else {
+          logger.warn( 'unknown geocoding error object:', err.constructor.name, err );
```

Comment:
i thought we were going to check for existance of err.contructor here? seems a bit optimistic to assume it's there